codegen-winter/src/air/mod.rs

```diff
@@ -109,19 +121,19 @@ let context = AirContext::new_multi_segment(
     new.line(context);
 
     // return initialized Self.
-    new.line("Self { context }");
+    new.line("Self { context, stack_inputs: public_inputs.stack_inputs, stack_outputs: public_inputs.stack_outputs, overflow_addrs: public_inputs.overflow_addrs }");
```

Comment:
Ah there's a tricky thing here - these things shouldn't be hard-coded. Yes, they are the public inputs that we are using in Miden, but the public inputs in the Air struct should only depend on the public inputs that are declared in the air file.